lib/archethic/p2p/message.ex

```diff
@@ -59,6 +59,7 @@ defmodule Archethic.P2P.Message do
   alias __MODULE__.RegisterBeaconUpdates
```

Comment:
A better way for import can be like this syntax : alias __MODULE__.{ModuleName1, ModuleName2, etc.}.
https://elixir-lang.org/getting-started/alias-require-and-import.html#multi-aliasimportrequireuse

Reply:
I think this can be improved for the entire codebase, so the scope of the PR would be too big I guess.
But yes you are right, it might be better.